dm/config/task.go

```diff
@@ -322,8 +330,8 @@ func (c *TaskConfig) Decode(data string) error {
 
 // adjust adjusts configs
 func (c *TaskConfig) adjust() error {
-	if len(c.Name) == 0 {
-		return terror.ErrConfigNeedUniqueTaskName.Generate()
+	if !c.verifyTaskName() {
+		return terror.ErrConfigTaskNameNotValid.Generate(fmt.Sprintf("task name %s not valid", c.Name))
```

Comment:
show the valid chars in the message?

or even how to handle compatibility issues?

Reply:
sure, [Done](https://github.com/pingcap/dm/pull/290/files#diff-b263153ff75f30e0c0f3fd51160160faR334)